app/assets/javascripts/modules/sticky-element-container.js

```diff
@@ -10,77 +10,77 @@
 (function (Modules, root) {
   'use strict'
 
-  var $ = root.$
-  var $window = $(root)
-
   Modules.StickyElementContainer = function () {
     var self = this
 
-    self._getWindowDimensions = function _getWindowDimensions () {
+    self.getWindowDimensions = function () {
       return {
-        height: $window.height(),
-        width: $window.width()
+        height: root.innerHeight,
+        width: root.innerWidth
       }
     }
 
-    self._getWindowPositions = function _getWindowPositions () {
+    self.getWindowPositions = function () {
       return {
-        scrollTop: $window.scrollTop()
+        scrollTop: root.scrollY
       }
     }
 
-    self.start = function start ($el) {
-      var $element = $el.find('[data-sticky-element]')
-      var _hasResized = true
-      var _hasScrolled = true
-      var _interval = 50
-      var _windowVerticalPosition = 1
-      var _startPosition, _stopPosition
+    self.start = function ($el) {
+      var el = $el[0]
+      var element = el.querySelector('[data-sticky-element]')
```

Comment:
Could these two lines be combined e.g. `var element = $el[0].querySelector('[data-sticky-element]')`?

Reply:
It can be a single line, but there are a couple of things that I thought made this approach better:

- we need a reference in `checkResize()` to the de-jQuery'd element - repeating `$el[0]` didn't seem quite right,
- my thinking was de-reference the jQuery object once to make it clear that from now on there is no jQuery, and 
- I'm guessing that at some point we'd want to pass into the `start()` a regular DOM element - having a variable set would make that change simpler.

Reply:
Ah of course, sorry hadn't noticed that when I made this comment.